Tests/test_PDB.py

```diff
@@ -1157,6 +1150,48 @@ def test_transform(self):
             for i in range(0, 3):
                 self.assertAlmostEqual(newpos[i], newpos_check[i])
 
+    def test_Vector(self):
+        """Test Vector object"""
+        v1 = Vector(0, 0, 1)
+        v2 = Vector(0, 0, 0)
+        v3 = Vector(0, 1, 0)
+        v4 = Vector(1, 1, 0)
+
+        self.assertEqual(calc_angle(v1, v2, v3), 1.5707963267948966)
+        self.assertEqual(calc_dihedral(v1, v2, v3, v4), 1.5707963267948966)
+        ref = refmat(v1, v3)
+        rot = rotmat(v1, v3)
+        self.assertEqual(ref[0].all(), numpy.array([1.0, 0.0, 0.0]).all())
```

Comment:
I think the ``.all()`` bits should be removed, note ``numpy.array([1.0, 0.0, 0.0]).all()`` u
is just ``False``.